addons/addon-environment-sc-api/packages/environment-sc-workflow-steps/lib/steps/stop-ec2-environment/stop-ec2-environment-sc.yml

```diff
@@ -0,0 +1,8 @@
+id: st-stop-ec2-environment-sc
+v: 1
+title: Stop EC2 Environment
```

Comment:
Optional: Add the phrase 'Service Catalog' in title and desc in these new files.